src/pybind/mgr/dashboard/frontend/cypress/integration/page-helper.po.ts

```diff
@@ -74,7 +74,7 @@ export abstract class PageHelper {
   }
 
   getTab(tabName: string) {
-    return cy.contains('.nav.nav-tabs li', new RegExp(`^${tabName}$`));
+    return cy.contains('.nav.nav-tabs li', tabName);
```

Comment:
Regexp is not required here?

Reply:
Yeah, i don't think so as it was too strict. For  e.g. in NFS-> Clients tab text is `Cients ({no_of_clients})` which doesn't fit the expected regex, so I thought better to just look for the required string.